### DISCUSSION TITLE
v2.15.4

### DISCUSSION THREAD

**Discussion by rabilrbl** [rabilrbl/jiotv_go#128] [Releases]
**Full Changelog**: https://github.com/rabilrbl/jiotv_go/compare/v2.15.3...v2.15.4

<hr /><em>This discussion was created from the release <a href='https://github.com/rabilrbl/jiotv_go/releases/tag/v2.15.4'>v2.15.4</a>.</em>

**Comment by iamswarnadeep** [comment]
I made a free tier EC2 instance on AWS in Mumbai location with ubuntu. Then I follow all the steps to install the script. It's perfectly working on localhost ip, but when I execute `./jiotv_go_full_name "publicIP:port"` [where, full_name is full file name, publicIP is my instance public ip like 64.0.125.219, port is 5001] then it's not working. So I check the log file, I saw an error message "main.go:89: failed to listen: listen tcp4 64.0.125.219:5001: bind: cannot assign requested address"

Can you help me to fix ?
